recipes/composable-kernel/recipe.yaml

```diff
@@ -0,0 +1,53 @@
+context:
+  name: composable-kernel
+  version: "7.0.2"
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ROCm/composable_kernel/archive/refs/tags/rocm-{{ version }}.tar.gz
+  sha256: b7293e3451750f606ab845585b3dd4eb4e185d4dda4a22290d73e8874a45a26b
+
+build:
+  number: 0
+  skip:
+    - not linux
+    - hip_compiler_version in (None, "None")
+
+requirements:
+  build:
+    - ${{ compiler('c') }}
+    - ${{ stdlib("c") }}
+    - ${{ compiler('cxx') }}
+    - ${{ compiler('hip') }}
+    - cmake
+    - ninja
+  host:
+    - rocm-core ${{ version }}.*
+    - rocm-cmake
+    - hip-devel
+  run:
+    - hip-runtime-amd
```

Comment:
This should be automatically set by the `strong` run_exports of `hip-clang_linux-64`, see https://github.com/traversaro/rock-the-conda/blob/main/recipes/hip/meta.yaml#L164-L166 .